cmd/crictl/container.go

```diff
@@ -221,6 +221,10 @@ var containerStatusCommand = cli.Command{
 			Name:  "output, o",
 			Usage: "Output format, One of: json|yaml|table",
 		},
+		cli.BoolFlag{
```

Comment:
I think for inspect, we should always set `verbose=true` in crictl.
`verbose=false` is for kubelet using CRI.

Reply:
@Random-Liu so you are saying you want a --quiet option for less verbose? or take out the option all-together? There were already other verbose flags so I figured that's what we would want with this verbose option.

Reply:
I feel like by default we should enable verbose, or else it's really troublesome to do -v every time. :)

As for whether we need a `quiet` option, I'm not sure.

Reply:
@Random-Liu I hear ya.. think it over no rush, easy enough to switch it to verbose by default, or quiet mode option. But I think when you see the verbose mode from the CRI-O team you'll be begging to turn down the volume :-) @runcom

Reply:
? Not following, what about CRI-O? We aren't even using that verbose stuff and we have no plan to output anything.
In any case, I'm in favor of going verbose by default like Lantao

Reply:
+1 of enabling verbose.

Reply:
I merely meant that CRI-O keeps a lot of data about the container to possibly use in a verbose output of status about the container.

Reply:
Yeah but that's exactly the verbose intention, despite what runtimes want to output

Reply:
I vote inspect to be verbose by default for crictl (and provide a silent flag to turn verbose off).
I vote to make the json format the default (and I don't care if you want to kill the table format or keep it as an option).

Reply:
done..